test/lit.cfg

```diff
@@ -833,3 +853,6 @@ config.substitutions.append(('%target-object-format', config.target_object_forma
 
 config.substitutions.append(('%target-resilience-test-wmo', config.target_resilience_test_wmo))
 config.substitutions.append(('%target-resilience-test', config.target_resilience_test))
+
+config.substitutions.append(('%target-llvm-profdata', os.path.join(llvm_tools_dir, 'llvm-profdata')))
+config.substitutions.append(('%target-llvm-cov', os.path.join(llvm_tools_dir, 'llvm-cov')))
```

Comment:
I don't think we need these. We should already find them via PATH, and they're unlikely to ever need a default set of parameters. (Also, there's nothing "target-" about them, so that should be left off whether or not we have a substitution.)